tests/QueueTest.php

```diff
@@ -1,8 +1,6 @@
 <?php
 
-use PHPUnit\Framework\TestCase;
-
-class QueueTest extends TestCase
+class QueueTest extends AbstractIndexingTestCase
```

Comment:
Why this class inherits AbstractIndexingTestCase when it's not really testing indexing?
If it just need it's dependencies I think the better approach would be to create AbstractTestCase, put all common stuff (like $indexPrefix) there and remove those stuff form AbstractIndexingTestCase.
Then make AbstractIndexingTestCase inherit from AbstractTestCase.
So we avoid use less code inherited everywhere. WDYT?